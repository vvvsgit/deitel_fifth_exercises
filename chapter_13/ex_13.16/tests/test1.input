500
1000

150
250

333
458

