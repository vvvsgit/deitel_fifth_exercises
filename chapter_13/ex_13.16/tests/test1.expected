
Account
The initial balance is 1000
Enter amount to withdraw: 
The balance after withdraw is 500
Enter amount to deposit: 
The balance after deposit is 1500

SavingsAccount
The initial balance is 1000
The balance after calculating interest is 1090
Enter amount to withdraw: 
The balance after withdraw is 940
Enter amount to deposit: 
The balance after deposit is 1190

CheckingAccount
The initial balance is 1000
Enter amount to withdraw: 
The balance after withdraw is 666.99
Enter amount to deposit: 
The balance after deposit is 1124.98


